### PR TITLE
KATI-89_Appointment-Request-List-UI

### PR DESCRIPTION
**Pull request by vtnlinh95**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/view/AnimationExecutor.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ */
+public class AnimationExecutor implements ExpandableStickyListHeadersListView.IAnimationExecutor {
+    private WeakHashMap<View,Integer> mOriginalViewHeightPool;
+    public AnimationExecutor(WeakHashMap<View,Integer> mOriginalViewHeightPool) {
+        this.mOriginalViewHeightPool = mOriginalViewHeightPool;
+    }
+
+    @Override
+    public void executeAnim(final View target, final int animType) {
+        if(ExpandableStickyListHeadersListView.ANIMATION_EXPAND==animType&&target.getVisibility()==View.VISIBLE){
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/AnimationExecutor.java]: re-format this file

**Comment by vtnlinh95** [comment]
@hieumai fixed

File: com.kms.cura.entity/src/com/kms/cura/entity/user/UserEntity.java [hunk 2]
```diff
@@ -1,5 +1,8 @@
 package com.kms.cura.entity.user;
 
+import java.util.List;
+
+import com.kms.cura.entity.AppointmentEntity;
```
Comment (hieumai) [inline com.kms.cura.entity/src/com/kms/cura/entity/user/UserEntity.java]: remove un-used imports

File: com.kms.cura.entity/src/com/kms/cura/entity/user/AdminUserEntity.java [hunk 3]
```diff
@@ -1,5 +1,9 @@
 package com.kms.cura.entity.user;
 
+import java.util.List;
+
+import com.kms.cura.entity.AppointmentEntity;
+
```
Comment (hieumai) [inline com.kms.cura.entity/src/com/kms/cura/entity/user/AdminUserEntity.java]: Remove un-used imports here too

File: app/src/main/java/com/kms/cura/view/activity/PatientViewActivity.java [hunk 4]
```diff
@@ -48,6 +48,11 @@ private void initFragments() {
         patientProfileFragment = new PatientProfileFragment();
         patientHealthTrachkerFragment = new HealthTrackerFragment();
         patientAppointmentFragment = new PatientAppointmentListFragment();
+        if (getIntent().getStringExtra(NAVIGATION_KEY) != null && getIntent().getStringExtra(NAVIGATION_KEY).equals(ConditionSymptomSearchActivity.TO_HEALTH_TRACKER)) {
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/PatientViewActivity.java]: extract getIntent().getStringExtra(NAVIGATION_KEY) into a variable for avoiding double method call

File: app/src/main/java/com/kms/cura/utils/CurrentUserProfile.java [hunk 5]
```diff
@@ -5,6 +5,10 @@
 import com.kms.cura.entity.user.PatientUserEntity;
 import com.kms.cura.entity.user.UserEntity;
 
+import java.sql.Date;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.List;
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/utils/CurrentUserProfile.java]: Remove unused imports

**Comment by vtnlinh95** [comment]
@hieumai fixed

**Comment by hieumai** [comment]
OK from me. @huuduy216  and @toannguyen71195  please start verifying on this

**Comment by hieumai** [comment]
@vtnlinh95  please rebase so I can merge

**Pull request merged**